### ISSUE TITLE
Computer crashes everytime.

### ISSUE THREAD

**Issue by chepe263** [zearp/Nucintosh#140]
Please provide the follow information:

- Steps to reproduce the issue -- this is crucial
 Open Chrome, play a youtube video. Computer Crashes
It also happens with safari.

- macOS version and optionally the build number
macOS Monterey 12.6.3 (21G419)

- Any changes you made to the config and/or EFI other than serials/etc
Disabled kext for wifi, bluetooth as per the instructions when you use an apple wifi card with an adapter

- Any logs related to the issue if applicable

I'm using a `NUC8CBEB`, bios version `BECFL257.86A.0089.2021.0621.1243`.

The computer crashes on Monterey, Ventura. It happens specially when I'm watching a youtube video using Google Chrome.

I disabled the M2 slot (no broadcom wifi temporarily), changed the sata drive. Changed the os (from ventura, back to monterey). No luck

I suspect the intel hd grapics are the real problem.

Do you have an idea on how to fix or diagnose the issue on this nuc?

**Comment by zearp** [comment]
Please share some of the crash/panic logs. Without those all we can do is guess.

**Comment by chepe263** [comment]
<details><summary>panic</summary>
<p>

panic(cpu 6 caller 0xffffff8001319339): [default.kalloc.4096]: element modified after free (off:2496, val:0x0000e50000000000, sz:4096, ptr:0xffffffa54de80000)
 2496: 0x0000e50000000000
 2512: 0x0000c50000000000
 2528: 0x0000c10000000000
 2544: 0x0000d10000000000
mp = 0xffffffa54d57bcf0, phys = 0x1f04ebcf0, prev (0xb: 0xff000000-0x100000000)
vp = 0xffffffa54f20a800, phys = 0x35dbc4800, prev (0xb: 0xff000000-0x100000000)
0xffffffa54d57a000: IUMAAAAAAC9UA2RldgBYA2RldmZcA3MAAABgAzBCVYBwB6D///90BwAAAgB4BwAAAgB8ByAAAACAByAAAACEBwAAAgCIBwAAAgCMB/8PAACQBwACAACUByAAAACYBwEAAACcB2oBAAC4B85LBwDAB2oBAADIBy5MBwDQBwAAQgDgB//////8Bz8AAAAQCJv///9ACJv///9ECMBhd+dICJb///9MCACaVE1QCKX///9UCAPfgxdYCA4AAABcCAIAAABgCP////+MCAEAAACQCMiyV02gCKX///+kCCiKV02oCKX///+sCA4AAACwCP/////ECEAd6QPICID////MCIDxxufQCJb////UCADxZE3YCKX////cCACgV07gCKX////kCCD8ZE3oCKX////sCPCoV034CKX////8CACpV00ICaX///8MCQCQkAQQCUAA3YAUCQAQAAAkCQAAEAAoCUweuQMwCf6JWAM4Cf6JWANACeMBAQBICZ6S1YVQCbCP1YVYCQUAAAFgCRoAAABkCQCQkARsCWFwZnN0CS9TeXOECXRlbS+ICVZvbHWMCW1lcy+QCVByZWKUCW9vdACYCS9TeXNsDXRlbS9wDVZvbHV0DW1lcy94DVByZWJ8DW9vdACADS9kZXaEDS9kaXOIDWsxczKMDQAA
0xffffffa54d57b000: nQAAAKYAAAC1AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAoPGouAEBAQAAEGIQgECQcAJBQQCITIZRZv2Cg6gAuAKBjIKqAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAoKgoKAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABBqLhofqAANABgEAAJkHCgUkEBAxKLUqNQqr4obroAqgCAqCAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADwHk2l////AAAABAABAAAAAEAAAwAAAAAQAAAAAEIAABFlTQCNeOcA/GRNjGbuB/C8V02gqFdNQB3pA4DxxucA8mRNAPkBThizV00AkJAEQADdgAAQAAAAABAATB65A/6JWAOoAAAAxo/VhQcAAAEaAAAAYXBmcy9TeXN0ZW0vVm9sdW1lcy9VcGRhdGUAAAAvU3lzdGVtL1ZvbHVtZXMvVXBkYXRlAC9kZXYvZGlzazFzNgCgH02l////AAAABAABAAAAAEAAAwAAAAAQAADnKw0AmSwNAAAAQgD/////ABVlTUCNeOcA82RNvLU7CBjHV03PWAAAQB3pA4DxxucA7mRNAEXsTSCoIE8ApyBPIO9kTQBKP1Ag6/5Ppf///wCQkARAAN2IABAAAAAAEABMHrkDGTRWAwPKIwA473iF6CN4hQQAAAEaAAAAAJCQBGFwZnMvU3lzdGVtL1ZvbHVtZXMvRGF0YQAAAODu4A4ODuDu7u6u504GDwAAAODg7g7goO7u7u7uDgAPAAGAEAAC/F86A2xuOaU7IAD/D1A6F/wPOJaXDjIowx4AIAQgAB+UPgCbW146DgjwPwSUjiz/A245pUMAAAAA
0xffffffa54d57c000: egAAAH8AAACIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgAoKCgoAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAyKDgZgeGAgNAiUAAAEHECgWGwSIREkCa9QoNIQMCCoAACKgKBgICAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACgoKCgAwEDAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAC9TeXN0ZW0vVm9sdW1lcy9EYXRhAC9kZXYvZGlzazFzMQEAAAAA8B9Npf///wAAAAQAAQAAAABAAAMAAAAAEAAAAGAAAEugDgAAAEIAAB9lTcCOeOcA72RNjUiZFADQGEwAAAACBQAAAPC8V01AcuiZwPDG5wBo7E0AV+xNaMdXTQAAUAQAAAgAAAQAAAACAAABAAA1HQAAAGF1dG9mcwAAL1N5c3RlbS9Wb2x1bWVzL0RhdGEvaG9tAAAvU3lzdGVtL1ZvbHVtZXMvRGF0YS9ob21lAG1hcCAAAAAABu7u4O7gAO7+7u5uDgAOCyAEIABq8fA/pQ8QAJtbXjoOCPA/pQfwP39bXjog410GXwAAAAAA
Panicked task 0xffffffa54cef8350: 3 threads: pid 1: launchd
Backtrace (CPU 6), panicked thread: 0xffffffa54c18d000, Frame : Return Address
0xfffffff2ce953060 : 0xffffff8000a7bebd mach_kernel : _handle_debugger_trap + 0x41d
0xfffffff2ce9530b0 : 0xffffff8000bde5e6 mach_kernel : _kdp_i386_trap + 0x116
0xfffffff2ce9530f0 : 0xffffff8000bcd953 mach_kernel : _kernel_trap + 0x4d3
0xfffffff2ce953140 : 0xffffff8000a1ba70 mach_kernel : _return_from_trap + 0xe0
0xfffffff2ce953160 : 0xffffff8000a7c28d mach_kernel : _DebuggerTrapWithState + 0xad
0xfffffff2ce953280 : 0xffffff8000a7ba46 mach_kernel : _panic_trap_to_debugger + 0x2b6
0xfffffff2ce9532e0 : 0xffffff80013148b3 mach_kernel : _panic + 0x84
0xfffffff2ce9533d0 : 0xffffff8001319339 mach_kernel : _panic_with_thread_context + 0x49b1
0xfffffff2ce953840 : 0xffffff8000ae6319 mach_kernel : _work_interval_port_type_render_server + 0x4f69
0xfffffff2ce9538a0 : 0xffffff8000a8ba50 mach_kernel : _ipc_thread_port_unpin + 0x340
0xfffffff2ce9538f0 : 0xffffff8000b5bc50 mach_kernel : _vm_page_delayed_work_finish_ctx + 0x1710
0xfffffff2ce953930 : 0xffffff8000b5a75b mach_kernel : _vm_page_delayed_work_finish_ctx + 0x21b
0xfffffff2ce953a60 : 0xffffff800109a4bf mach_kernel : _ubc_create_upl_kernel + 0x10f
0xfffffff2ce953a90 : 0xffffff8000d0d01e mach_kernel : _cluster_write_ext + 0x285e
0xfffffff2ce953b30 : 0xffffff8000d0cd26 mach_kernel : _cluster_write_ext + 0x2566
0xfffffff2ce953c70 : 0xffffff8000d12672 mach_kernel : _cluster_push_err + 0x232
0xfffffff2ce953cf0 : 0xffffff8003d54c90 com.apple.filesystems.apfs : _apfs_fsync_internal + 0xe8
0xfffffff2ce953d70 : 0xffffff8003d9891e com.apple.filesystems.apfs : _apfs_sync_callback + 0x52
0xfffffff2ce953da0 : 0xffffff8000d1a83a mach_kernel : _vnode_iterate + 0x28a
0xfffffff2ce953e50 : 0xffffff8003d93474 com.apple.filesystems.apfs : _apfs_vfsop_sync + 0x82
0xfffffff2ce953ea0 : 0xffffff8000d2ef3e mach_kernel : _sync + 0x9e
0xfffffff2ce953ec0 : 0xffffff8000d1a4dc mach_kernel : _vfs_iterate + 0x1fc
0xfffffff2ce953f30 : 0xffffff8000d2eeb7 mach_kernel : _sync + 0x17
0xfffffff2ce953f40 : 0xffffff80011869cb mach_kernel : _unix_syscall64 + 0x1fb
0xfffffff2ce953fa0 : 0xffffff8000a1c236 mach_kernel : _hndl_unix_scall64 + 0x16
      Kernel Extensions in backtrace:
         com.apple.filesystems.apfs(1934.141.2)[1301B918-E7B9-3B7D-B547-8531187752C8]@0xffffff8003d0b000->0xffffff8003e89fff
            dependency: com.apple.driver.AppleEFINVRAM(2.1)[D81CCB95-87B9-39B3-9DD0-CF8BFE441FBD]@0xffffff8001fb2000->0xffffff8001fbbfff
            dependency: com.apple.driver.AppleEffaceableStorage(1.0)[EB0B720C-C8E6-3516-B040-018D9D2B106C]@0xffffff8001fc8000->0xffffff8001fcdfff
            dependency: com.apple.iokit.CoreAnalyticsFamily(1)[66637B5A-DB83-3B8E-9B27-E86306FC3B42]@0xffffff80024d3000->0xffffff80024dafff
            dependency: com.apple.iokit.IOStorageFamily(2.1)[B70CE90E-3064-37AD-B553-651F18B761B3]@0xffffff800377b000->0xffffff8003791fff
            dependency: com.apple.kec.corecrypto(12.0)[E6EED25F-C823-36DE-837E-02C891ADFBF7]@0xffffff8003ec3000->0xffffff8003f43fff
            dependency: com.apple.security.AppleImage4(4.2.0)[535F9FB3-1ADC-32FC-8FC0-F01807C8BB2E]@0xffffff800204f000->0xffffff8002067fff

Process name corresponding to current thread (0xffffffa54c18d000): launchd
Boot args: -v keepsyms=1 chunklist-security-epoch=0 -chunklist-no-rev2-dev

Mac OS version:
21G419

Kernel version:
Darwin Kernel Version 21.6.0: Mon Dec 19 20:44:01 PST 2022; root:xnu-8020.240.18~2/RELEASE_X86_64
Kernel UUID: 33C30416-3302-3FFF-9686-A557DED28C5B
KernelCache slide: 0x0000000000800000
KernelCache base:  0xffffff8000a00000
Kernel slide:      0x0000000000810000
Kernel text base:  0xffffff8000a10000
__HIB  text base: 0xffffff8000900000
System model name: Macmini8,1 (Mac-7BA5B2DFE22DDD8C)
System shutdown begun: NO
Panic diags file available: YES (0x0)
Hibernation exit count: 0

System uptime in nanoseconds: 362960070424
Last Sleep:           absolute           base_tsc          base_nano
  Uptime  : 0x00000054821b2c59
  Sleep   : 0x0000000000000000 0x0000000000000000 0x0000000000000000
  Wake    : 0x0000000000000000 0x00000006f0754b6c 0x0000000000000000
Compressor Info: 0% of compressed pages limit (OK) and 0% of segments limit (OK) with 0 swapfiles and OK swap space
Zone info:
  Zone map: 0xffffff907f7ac000 - 0xffffffb07f7ac000
  . PGZ   : 0xffffff907f7ac000 - 0xffffff90817ad000
  . VM    : 0xffffff90817ad000 - 0xffffff954dfac000
  . RO    : 0xffffff954dfac000 - 0xffffff96e77ac000
  . GEN0  : 0xffffff96e77ac000 - 0xffffff9bb3fac000
  . GEN1  : 0xffffff9bb3fac000 - 0xffffffa0807ac000
  . GEN2  : 0xffffffa0807ac000 - 0xffffffa54cfac000
  . GEN3  : 0xffffffa54cfac000 - 0xffffffaa197ac000
  . DATA  : 0xffffffaa197ac000 - 0xffffffb07f7ac000
  Metadata: 0xffffffb0937bc000 - 0xffffffb0b37bc000
  Bitmaps : 0xffffffb0b37bc000 - 0xffffffb0b97bc000



---------------------------------------------------

panic(cpu 3 caller 0xffffff8003dcdd43): Kernel trap at 0xffffff800422d279, type 14=page fault, registers:
CR0: 0x0000000080010033, CR2: 0x0000040000000020, CR3: 0x00000004a942a00c, CR4: 0x00000000003626e0
RAX: 0x0000000000014c2b, RBX: 0xffffff800464a458, RCX: 0xffffff8b6f045400, RDX: 0x000000000000003f
RSP: 0xffffffbdc7643cf0, RBP: 0xffffffbdc7643dc0, RSI: 0xffffff8b6ee88ba0, RDI: 0xffffff8b6ee88c78
R8:  0xffffff800464a230, R9:  0xffffff8b6ee88ba0, R10: 0xffffffbdc7643e28, R11: 0x0000000000000048
R12: 0xffffffbdc7643e28, R13: 0xffffff8b6f05be80, R14: 0x0000000000000000, R15: 0x0000040000000000
RFL: 0x0000000000010206, RIP: 0xffffff800422d279, CS:  0x0000000000000008, SS:  0x0000000000000010
Fault CR2: 0x0000040000000020, Error code: 0x0000000000000000, Fault CPU: 0x3, PL: 0, VF: 1

Panicked task 0xffffff950850b3b0: 12 threads: pid 142: AirPlayXPCHelper
Backtrace (CPU 3), panicked thread: 0xffffff8b6eef7540, Frame : Return Address
0xffffffbdc76436a0 : 0xffffff8003c7bebd mach_kernel : _handle_debugger_trap + 0x41d
0xffffffbdc76436f0 : 0xffffff8003dde5e6 mach_kernel : _kdp_i386_trap + 0x116
0xffffffbdc7643730 : 0xffffff8003dcd953 mach_kernel : _kernel_trap + 0x4d3
0xffffffbdc7643780 : 0xffffff8003c1ba70 mach_kernel : _return_from_trap + 0xe0
0xffffffbdc76437a0 : 0xffffff8003c7c28d mach_kernel : _DebuggerTrapWithState + 0xad
0xffffffbdc76438c0 : 0xffffff8003c7ba46 mach_kernel : _panic_trap_to_debugger + 0x2b6
0xffffffbdc7643920 : 0xffffff80045148b3 mach_kernel : _panic + 0x84
0xffffffbdc7643a10 : 0xffffff8003dcdd43 mach_kernel : _sync_iss_to_iks + 0x2c3
0xffffffbdc7643b90 : 0xffffff8003dcda26 mach_kernel : _kernel_trap + 0x5a6
0xffffffbdc7643be0 : 0xffffff8003c1ba70 mach_kernel : _return_from_trap + 0xe0
0xffffffbdc7643c00 : 0xffffff800422d279 mach_kernel : _kevent_register + 0x209
0xffffffbdc7643dc0 : 0xffffff8004230f1a mach_kernel : _kevent_id + 0x39a
0xffffffbdc7643ec0 : 0xffffff8004230d16 mach_kernel : _kevent_id + 0x196
0xffffffbdc7643f40 : 0xffffff80043869cb mach_kernel : _unix_syscall64 + 0x1fb
0xffffffbdc7643fa0 : 0xffffff8003c1c236 mach_kernel : _hndl_unix_scall64 + 0x16

Process name corresponding to current thread (0xffffff8b6eef7540): AirPlayXPCHelper
Boot args: -v keepsyms=1 chunklist-security-epoch=0 -chunklist-no-rev2-dev

Mac OS version:
21G419

Kernel version:
Darwin Kernel Version 21.6.0: Mon Dec 19 20:44:01 PST 2022; root:xnu-8020.240.18~2/RELEASE_X86_64
Kernel UUID: 33C30416-3302-3FFF-9686-A557DED28C5B
KernelCache slide: 0x0000000003a00000
KernelCache base:  0xffffff8003c00000
Kernel slide:      0x0000000003a10000
Kernel text base:  0xffffff8003c10000
__HIB  text base: 0xffffff8003b00000
System model name: Macmini8,1 (Mac-7BA5B2DFE22DDD8C)
System shutdown begun: NO
Panic diags file available: YES (0x0)
Hibernation exit count: 0

System uptime in nanoseconds: 455623817934
Last Sleep:           absolute           base_tsc          base_nano
  Uptime  : 0x0000006a154bc1b8
  Sleep   : 0x0000000000000000 0x0000000000000000 0x0000000000000000
  Wake    : 0x0000000000000000 0x00000006f101bd46 0x0000000000000000
Compressor Info: 0% of compressed pages limit (OK) and 0% of segments limit (OK) with 0 swapfiles and OK swap space
Zone info:
  Zone map: 0xffffff8039e6d000 - 0xffffffa039e6d000
  . PGZ   : 0xffffff8039e6d000 - 0xffffff803be6e000
  . VM    : 0xffffff803be6e000 - 0xffffff850866d000
  . RO    : 0xffffff850866d000 - 0xffffff86a1e6d000
  . GEN0  : 0xffffff86a1e6d000 - 0xffffff8b6e66d000
  . GEN1  : 0xffffff8b6e66d000 - 0xffffff903ae6d000
  . GEN2  : 0xffffff903ae6d000 - 0xffffff950766d000
  . GEN3  : 0xffffff950766d000 - 0xffffff99d3e6d000
  . DATA  : 0xffffff99d3e6d000 - 0xffffffa039e6d000
  Metadata: 0xffffffbdc79b5000 - 0xffffffbde79b5000
  Bitmaps : 0xffffffbde79b5000 - 0xffffffbded9b5000





</p>
</details>

**Comment by zearp** [comment]
Thanks for sharing, unfortunately they're not very helpful and you're the first person to report this. Some of your crashes don't have a kernel extension list, if you have more crash logs check the loaded extensions for ones that keep returning. Though I don't think it is caused by a bad extension/kext. The ones in your logs are system kexts, not ones in the EFI.

I don't have any Monterey installed to try and reproduce it on but I do use Brave on Ventura and Sonoma without issues. Which is also based on Chromium like Chrome.

I did notice Airplay in one of those logs, but it could be unrelated.

I assume playing videos in Chrome causes no issues in Windows or Linux. Not sure if you tried it, or tried other video related stuff on macOS. If it's only Chrome you could try a different version before doing any of the above. Or maybe try Firefox, Edge or Brave too. If it only happens in Chrome then surely this is Chrome issue somehow. But I don't know what you have tried and tested.

What I would suggest is to make a bootable clone or TimeMachine backup and then do a clean install including resetting nvram and formatting your disk. Then in the cleanly installed macOS install Chrome and see if it happens again. If not you can import your apps and setting from the backup using the Migration tool and try again. 

If it happens again I can try it on my test NUC but it will take some time as I would need to do a clean install and find a disk somewhere I can use for that test. A lot of my spare disks are currently in use for Sonoma testing and my own backups. Also even if I can reproduce it I don't know if I can fix it and personally I never run into anything like this and I use several NUCs 24/7 with macOS. Also nobody else reported any similar issues. For a long time there's been no crash or panic issues at all. Specially on stock NUCs things just work.

For me to be able to help the issue has to be reproducible, thats why I'm asking to try a clean a install. It is a bit tedious but its the only way to fix these kind of things. I have no problem trying to reproduce your issue but you have make sure it is reproducible on your end first. If every time you do a fully clean install the same problem happens then its reproducible. Then anyone else doing the same on the same machine should get the same result. If they don't then you may have an h/w issue.

As long as you keep a good backup there's never a need to setup everything up from scratch on macOS. The migration wizard can import all the things from a backup. So the only thing it takes is some time to wipe, reinstall and test. I do this for every new release of the EFI, a clean install and some standard tests to make sure it works.

Wipe the disk, wipe nvram, do a clean install and only after you tried Chrome to see if the problem persists import your apps/settings. It is possible the crash won't happen before restoring your settings but will after, which would imply the issue is caused by a user setting. You could test that right now by making a new user and see if the crash happens a new user. If it doesn't happen as new user then it's a settings/software issue on your user account.

Long story short, because the panics have little in common with each other and you're the first person to report these kind of crashes it is most likely not caused by the EFI but something else. What that something else is requires some troubleshooting.

Lots of things to try. Sorry there's no easy clear cut solution for this. It is too vague at this point. We need to know if the same issue happens in other browsers and other operating systems, if it does it could indicate something else like ram issues. thjere are simply too many variables right now.

**Comment by chepe263** [comment]
I did a clean install of Monterey before making this issue. I had a 1TB WD sata ssd and changed it to a smaller 250GB Sillicon Power sata ssd.

The new install had only iCloud login and a fresh copy of chrome.

I played a video (from yt) in Safari and it also crashed. 

I was wondering how you make the ssdts and how you make them different from the dortania guide.

**Comment by zearp** [comment]
Well if it also happens in Safari then it sounds like an upcoming h/w issue, maybe a ram stick that is failing. What happens in Linux or Windows? If this was a common problem with the EFI lots more issues would be created with crashes like you have. I would've also ran into it because I reinstall and mess about a lot with several NUCs all running macOS. Sicne I can't reproduce your issue there is nothing else I can do unfortunately. I can only help with issues caused the EFI and I don't mind trying to help with unrelated issues such as this but in your case I can't do much other than guess which is not very helpful.

**Comment by chepe263** [comment]
using a nvme to usb adapter, somehow the system is a little more stable 🤔
